scripts/pkg_in_pipe/pkg_in_pipe.py

```diff
@@ -129,18 +139,21 @@ def print_table_line(out, build, link, issues, built_by, prs: list[PullRequest],
             </li>'''
         for i in issues
     ])
+    warn_pr = "<span class='has-tooltip'><span class='tooltip p-1 rounded bg-blue-500 text-gray-900'>Pull request not listed in any of the related cards</span>⚠️</span>"  # noqa
     prs_content = '\n'.join([
         f'''<li>
                 <a target="_blank" class="font-medium text-blue-600 dark:text-blue-500 hover:underline"
                    href="{pr.html_url}">{pr.title} #{pr.number}
-                </a> {"" if issues_have_link(issues, pr.html_url) else "⚠️"}
+                </a> {"" if issues_have_link(issues, pr.html_url) else warn_pr}
             </li>'''
         for pr in prs
     ])
+    warn_build = "<span class='has-tooltip'><span class='tooltip p-1 rounded bg-blue-500 text-gray-900'>Build not listed in any of the related cards</span>⚠️</span>"  # noqa
+    warn_maintainer = "<span class='has-tooltip'><span class='tooltip p-1 rounded bg-blue-500 text-gray-900'>Package has no maintainer</span>⚠️</span>"  # noqa
```

Comment:
Just to avoid having people start claiming that XCP-ng has packages without maintainers, I'd rephrase as "Maintainer information missing".